src/frontend-pwa/src/components/Header/Header.tsx

```diff
@@ -9,26 +14,20 @@ import {
   Heading,
   Banner,
   Image,
-  StyledSettingsButton,
-  StyledIcon,
 } from './header.styles';
 
 export default function Header() {
   return (
     <HeaderWrapper>
       <Banner>
-        <Image href="https://gov.bc.ca">
-          <img src={logo} alt="Go to the Government of British Columbia website" />
-        </Image>
+        <a href="/">
+          <Image src={logo} alt="Go to the Home page" />
+        </a>
         <Heading>Wayfinder</Heading>
       </Banner>
       <Link to="/settings">
-        <StyledSettingsButton
-          type="button"
-          aria-label="Settings button"
-        >
-          <StyledIcon src={gear} alt="Settings" />
-        </StyledSettingsButton>
+        <SettingsNavButton />
+        <BackNavButton />
       </Link>
```

Comment:
Both your Settings button and Back button link to the /settings page

Reply:
@LocalNewsTV Done!